pwiz_tools/Skyline/Test/TextUtilTest.cs

```diff
@@ -96,5 +97,155 @@ private IEnumerable<string> AddSuffix(string[] baseStrings, string fixText, int
+                "RHuC2","rhvbR","RiJUy","rKBjB","rkbvv","rkfQI","rL7oT","rLL6t","rlw8H","rn8i8","RNLar","ROcwU",
+                "rPeOk","-RrlA","RRS7e","RtIFu","RTl2L","Rur-i","rveQ3","rvQfl","rVxej","-RWWt","Ry3v7","rYq7J",
+                "S_Cag","s0hGb","S0Uut","S1FBc","S2mEF","S2QGv","s3HBn","S6vzd","S9B8l","sAU6F","SBrbJ","sBrJT",
+                "s-d7C","-sfqs","sI5SG","SIW2y","six2m","SkGrK","SM_9d","SMP4O","sna_M","soBNX","SPkR6","sprQw",
+                "sqVpD","srzYs","STpr1","SUGz6","Svw8U","SWIW9","SwvOY","syhEY","SyjtP","szFB5","szMUT","T_gxI",
+                "T1d-m","T3KhJ","T3oqE","T5Hgq","t7aSK","t9YtG","T093H","TAAbp","TachS","taDt_","tchMy","Tgmug",
+                "tgXKx","TH_Wu","ThIN1","thxs7","tiZNw","TLZRY","tNe7F","TnG1f","tNJV-","tP0lD","tPNqI","tPVBw",
+                "tSW4s","ttNVz","ttVVL","TVB1P","TVzX7","tWw6J","Tx214","U_aDj","u0sZU","u1LRx","U4sCZ","u5oh0",
+                "u5yMi","u17mQ","UAERJ","UaXHu","Ub2aG","ubMlW","u-d09","ue7uO","uEd03","UEgaC","uf09S","uFkkO",
+                "-uG-g","UIQF2","UJt7P","UjY-6","uk2zH","UN_mf","UNGYS","uNZ49","UoTD9","uP0_z","UpBUn","UPLhZ",
+                "UPTFO","uREtq","urPvG","uRRfr","usQpl","Usuof","USZgP","USZi3","Ut8mt","uuL71","uus4v","uw7Qx",
+                "UwQiJ","uXJ3Z","uYGFC","uygkU","Uznaa","uz-Oq","v0Bii","V3c_T","vAA99","vBJBq","vDfH7","vdsCT",
+                "Ve6Ta","ve7Ml","VEoAO","Vf6ec","vfINc","VipmQ","VKQTr","vKRM9","vlpJ-","Vm7DS","vNfbo","vnOtG",
+                "vnyz6","Vo1-6","vp_WA","vp4ad","vPjSH","vpr5l","vqlAw","vscw3","vssyk","VSxLB","vt3g_","vTjAk",
+                "vy-8M","Vz811","vZkWO","W_17N","W_zAH","w0W0o","W2ffg","w3bah","w5m84","W5TN-","W7_Gg","W8XKm",
+                "w19Ht","w430-","WadIY","waINu","w-ayL","wAYVn","Wb1iY","wb9eC","wcnac","Wc-Se","WCwth","Wcyyt",
+                "wD-Mr","WEMDm","WfG49","wgnCt","wH7z5","Whe2M","wHeOF","WHs9b","wj-Sy","wkzjo","wlz9S","WObZ2",
+                "WohAe","woie2","WOrKF","wp9GI","wSEvL","wSmeO","WstBf","wTpm1","WvIMM","WXnQH","Wzi2E","X_-fO",
+                "x_wnm","X7bAs","x8W-O","X78mS","xcEvh","xcmmi","xcqX_","xdiAw","xdnqq","XEgEz","xESjA","xfkcU",
+                "XfWhr","XfY9w","xgThP","-XI_N","XiSQ3","XJ0Ij","xJl17","XjWWj","XlGMw","xlt5k","XMgjD","XPCHC",
+                "XPxAQ","XQkWA","xRiK2","XSkvk","XUO_k","XWPlc","Xx0hO","XXA97","XxgDy","XZ_VJ","xZfiP","Y6Egb",
+                "Y6KYN","y7C8P","y7VP8","Y8P0V","Y8tqk","YBhcq","YbyUR","yE59F","yewLE","YFavT","yFJtl","YFnAx",
+                "yIaqR","YJ_Bw","ykUfT","Ylp6y","ymA2t","yPNFs","-yqQU","YS_68","YsfqU","YtEmi","ytYCY","YtZZ8",
+                "yUzgJ","YZ09K","yZeZY","Yzlgb","Z-0NM","z1hKB","z7DVy","Z8FV9","Z8qKD","Z96lA","zAu-r","ZBE3X",
+                "ZbPja","zbUz-","ZDjTb","Zdrnb","zFjSv","zFunl","zIW8u","zjUA6","zL3Mn","ZLImV","ZLpfb","zMVI0",
+                "zOpUQ","Zuzaq","ZvFIb","zXdQ1","zxWHd","zz0Pq","ZzIxa","zzlPp","ZZPVw"
+            };
+
+

```

Comment:
Not a single comment anywhere in this code. It's possible to over-comment, of course, but a quick sketch of the logic of the test is a good idea

// Take a known-sorted list, randomize it, make sure that our sort algorithm puts it back in order again